### PR TITLE
[FEATURE] Adding the scraping of the reviews sections for an anime

### PR DESCRIPTION
**Pull request by JustalK**
Hello,

I added the reviews scraping following the same structure of the episodes scraping.
Since the number of comments is not a finished number, I added two parameter for creating a pagination system in case.
`limit` : Allow you to limit the result to a certain number of reviews
`skip` : Allow you to skip a number of review
It was what i needed the most for one of my personal tool.

I added the tests to the implementation made (95% cov).
I also added the documentation and commented my code in the hard section.
I also edited a bit the package.json for fixing this => https://github.com/Kylart/MalScraper/issues/36#issue-681196131

Voila ;)

PS: I saw the modifications of the pull request but it has not move for quite a while (~6 months), the developer is not active anymore and I needed it now.

### CODE REVIEW

**Comment by JustalK** [comment]
If any question or additional thing to do, ask me ;)

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "Scrap everything you can from MyAnimeList.net",
   "main": "src/index.js",
   "scripts": {
-    "test": "npm run lint && nyc ava --verbose --serial test/*.test.js",
+    "test": "npm run lint && nyc ava --verbose --timeout=1m --serial test/*.test.js",
+    "test-no": "nyc ava --verbose --timeout=1m --serial test/reviews.test.js",
```
Comment (Kylart) [inline package.json:8]:
```suggestion
    "test-no": "nyc ava --verbose --timeout=1m --serial test/*.test.js",
```

File: src/reviews.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    result.push({
+      author: $($(this).find('.spaceit td:nth-child(2) a')['0']).text().trim(),
+      date: malDateToJsDate($($(this).find('.spaceit .mb8 div')['0']).text().trim()),
+      seen: $(this).find('.spaceit .mb8 .lightLink').text().trim(),
+      note_overall: malNumberToJsNumber($(notes).find('tr:nth-child(1) td:nth-child(2)').text().trim()),
+      note_story: malNumberToJsNumber($(notes).find('tr:nth-child(2) td:nth-child(2)').text().trim()),
+      note_animation: malNumberToJsNumber($(notes).find('tr:nth-child(3) td:nth-child(2)').text().trim()),
+      note_sound: malNumberToJsNumber($(notes).find('tr:nth-child(4) td:nth-child(2)').text().trim()),
+      note_character: malNumberToJsNumber($(notes).find('tr:nth-child(5) td:nth-child(2)').text().trim()),
+      note_enjoyment: malNumberToJsNumber($(notes).find('tr:nth-child(6) td:nth-child(2)').text().trim()),
```
Comment (Kylart) [inline src/reviews.js:48]: The rest of the library uses cameCase. Would it be ok to change the snake_case to CamelCase?

Also, I think that something like that would be kinda easier to use:
```javascript
note: {
  overall: malNumberToJsNumber($(notes).find('tr:nth-child(1) td:nth-child(2)').text().trim()),
  story: malNumberToJsNumber($(notes).find('tr:nth-child(2) td:nth-child(2)').text().trim()),
  ...
}
```

Comment (JustalK) [reply src/reviews.js]: Ha yes, I did not pay attention to that.
I made the change and remove the `note_`
I also change the package json to all tests :  *.tests

**Comment by Kylart** [comment]
Thanks for the PR, this will be deployed soon with tag 2.8.0!

**Pull request merged**